### DISCUSSION TITLE
2023-10-11

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#631] [daily_meeting]
### 오늘의 할 일
- 프론트엔드
  - 오전 강의

- 백엔드
  - 시간이 안맞는 문제 해결
  - 성능 개선 논의

- 공통
  - 오후에 폴링 회의

### 나중에 할 일
- 폴링 구현 목요일까지 가능할듯
- 회원 탈퇴(폴링, 배포 이후에)

### 개인 일정
- 노아 근로 (5-6)
- 마코, 룩소 근로 (3-4)

**Comment by aak2075** [comment]
### 체크인

10점 - 아침에 2문제나 풀어서 기분이 좋네요

### 체크아웃

10점 - 시간이 참 빠르네요

**Comment by nlom0218** [comment]
### 체크인

10점 - 저도 올만에 코테를 풀어서 기분이 좋아요. 

### 체크아웃

10점 - 이전에 oauth을 확장가능하게 코드를 작성했더니 오늘 카카오 oauth 작업이 너무 수월했어요!

**Reply by aak2075** [reply]
멋져요

**Comment by woo-jk** [comment]
### 체크인
8점 - 미션 시작합니다

### 체크아웃
9점 - 미션 끝났습니다

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 약 먹으니 훨씬 나은 컨디션 같습니다. 차근차근 취준하니까 좀 다시 안정을 찾아가는 기분이에요

### 체크아웃
6점 - 아 미션 빨리... ㅋㅋㅋ 멘탈 석 나가네요

**Reply by aak2075** [reply]
건또건이죠

**Reply by MoonJeWoong** [reply]
건강! 건강! 건강!

**Comment by jaehee329** [comment]
### 체크인
9점 - 곧 두문제 풀거같아요

### 체크아웃
9점 - 열심히 살았어요 우리

**Comment by woosung1223** [comment]
### 체크인
8점 - 할게 많군요..

### 체크아웃
8점 - 오늘도 다사다난 했군요..

### 체크아웃(허브)
10점 - 테오가 리뷰어라 10점입니다.

**Comment by yeopto** [comment]
### 체크인

8점 - 쏘쏘합니다.

### 체크아웃

9점 - 미션 제출해서 홀가분하네용
